pydantic/types.py

```diff
@@ -187,6 +188,26 @@ def validate(cls, value: str) -> str:
         return validate_email(value)[1]
 
 
+class StrictBool(int):
+    """
+    StrictBool to allow for bools which are not type-coerced.
+    """
+
+    @classmethod
+    def __get_validators__(cls) -> 'CallableGenerator':
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, value: str) -> bool:
+        """
+        Ensure that we only allow bools.
+        """
+        if isinstance(value, bool):
```

Comment:
I thought we were going to allow `0`, `1` and some string values.

Reply:
I thought this was a `StrictBool` and that the conversation about the other bool was for the more loose types?

TBH this is the behaviour that I need, and `Strict` implies exactly that.

Reply:
Okay